rmw/include/rmw/rmw.h

```diff
@@ -803,6 +803,41 @@ RMW_WARN_UNUSED
 rmw_ret_t
 rmw_publisher_assert_liveliness(const rmw_publisher_t * publisher);
 
+/// Waits until all published message data were acknowledged or timeout.
```

Comment:
nit: minimal rewording

```suggestion
/// Wait until all published message data is acknowledged or until the specified timeout elapses.
```